images/assets/add_signing_service.sh

```diff
@@ -3,7 +3,7 @@
 /usr/bin/wait_on_postgres.py
 /usr/bin/wait_on_database_migrations.sh
 
-if [ -n "${PULP_SIGNING_KEY_FINGERPRINT}" ]; then
+if [ -n "${PULP_SIGNING_KEY_FINGERPRINT-}" ]; then
```

Comment:
Isn't that:
```suggestion
if [ -n "${PULP_SIGNING_KEY_FINGERPRINT:-}" ]; then
```

Reply:
Adding the colon makes it also a nullity check. Do you think we need to check also if the value is null, since the default I am setting it to is null? (I am actually not sure how null works in bash land)

https://devhints.io/bash#parameter-expansions

Reply:
Well, I'd say if the variable was not set or set to "" it is safe to assume we cannot configure a signing service from it.
But honestly i didn't know about "-" alone.
But now reading up, i think both produce the same result here, as we replace with "".